RELEASE-NOTES.txt

```diff
@@ -1,5 +1,7 @@
 15.5
 -----
+ * [***] Block Editor: New feature for WordPress.com and Jetpack sites: auto-complete username mentions. An auto-complete popup will show up when the user types the @ character in the block editor.
```

Comment:
Interesting. I assume this got here as a result of 27630f4. I assume `develop` was merged into here to fix some merge conflicts.

Reply:
@mokagio This was added in https://github.com/wordpress-mobile/WordPress-iOS/pull/14569 which you can find it through the git blame view here: https://github.com/wordpress-mobile/WordPress-iOS/blame/merge/15.4-final-into-develop/RELEASE-NOTES.txt#L3. This is because the feature got enabled in the `release/15.4` branch for WPiOS and WPAndroid, but since it was late in the release cycle, the announcement got postponed to `15.5`.